crates/stackable-operator/src/eos/mod.rs

```diff
@@ -125,37 +127,37 @@ impl EndOfSupportChecker {
             // TODO: Add way to stop from the outside
             // The first tick ticks immediately.
             interval.tick().await;
+            let now = Utc::now();
+
             tracing::info_span!(
                 "checking end-of-support state",
                 eos.interval = self.interval.to_string(),
+                eos.now = now.to_rfc3339(),
             );
 
             // Continue the loop and wait for the next tick to run the check again.
-            if !self.is_eos() {
+            if now <= self.eos_datetime {
                 continue;
             }
 
-            self.emit_warning();
+            self.emit_warning(now);
         }
     }
 
     /// Emits the end-of-support warning.
     #[instrument(level = Level::DEBUG, skip(self))]
-    fn emit_warning(&self) {
+    fn emit_warning(&self, now: DateTime<Utc>) {
+        let built_datetime = self.built_datetime.to_rfc3339();
+        let build_age = Duration::try_from(now - self.built_datetime)
+            .expect("time delta of now and built datetime must not be less than 0")
+            .to_string();
+
         tracing::warn!(
-            eos.date = self.datetime.to_rfc3339(),
-            "the operator reached end-of-support"
+            eos.built.datetime = built_datetime,
+            eos.build.age = build_age,
+            "This operator version was built on {built_datetime} ({build_age} ago) and may has reached end-of-support. \
```

Comment:
Did you accidentally revert https://github.com/stackabletech/operator-rs/pull/1103#discussion_r2398980404?
```suggestion
            "This operator version was built on {built_datetime} ({build_age} ago) and may have reached end-of-support. \
```